listener/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/kyma-project/runtime-watcher/listener
 
 go 1.19
 
+replace github.com/kyma-project/runtime-watcher/kcp => ../kcp
```

Comment:
Please do not forget to open a follow-up-PR removing this and updating the kcp pkg to the latest. To avoid such, you could also open one PR for the kcp changes and another one for using them, just for the future

Reply:
Follow-up PR: https://github.com/kyma-project/runtime-watcher/pull/54